experimental/addons/README.md

```diff
@@ -0,0 +1,19 @@
+# KubeVela experimental addons
+
+This dir is KubeVela official experimental addon-registry which will contain hasn't verified stable KubeVela addon.
+
+Addon files in this dir files will be synced to alibaba-oss but will not be set in KubeVela by default.
+
+You can add this addon registry by vela cli then use these addons.
+
+```shell
+$ vela addon registry add experimental --type=git --gitUrl=https://github.com/oam-dev/catalog/ --path=experimental/addons
```

Comment:
I think  we should also have an OSS bucket address here?

Reply:
After this PR is merged https://github.com/oam-dev/kubevela/pull/2907

Reply:
Cli have support addon registry oss path by https://github.com/oam-dev/kubevela/pull/2976.